src/components/BrowserApp.test.jsx

```diff
@@ -1,8 +1,8 @@
 import React from 'react';
-import TestUtils from 'react-dom/test-utils';
+import ShallowRenderer from 'react-test-renderer/shallow';
```

Comment:
you can ignore this change - just fixing a deprecation warning in tests now that the shallow renderer has been moved to a different package.